app/assets/javascripts/shared/requests/serializer.js

```diff
@@ -0,0 +1,11 @@
+const MAPPER = {
```

Comment:
Should you call this file `user_serializer.js`? Don't think `MAPPER` is very clear either.